applications/serial_lte_modem/src/main.c

```diff
@@ -366,6 +374,142 @@ static void indicate_wk(struct k_work *work)
 	indicate_stop();
 }
 
+#ifdef CONFIG_LWM2M_CARRIER
+void nrf_modem_recoverable_error_handler(uint32_t err)
+{
+	printk("Modem library recoverable error: %u\n", (unsigned int)err);
+}
```

Comment:
This does not exist anymore

Reply:
Should also be removed from `lwm2m_carrier`

Reply:
Thanks 🙂